pymc/model.py

```diff
@@ -983,6 +907,32 @@ def unobserved_value_vars(self):
 
         return vars + untransformed_vars + deterministics
 
+    @property
+    def disc_vars(self):
```

Comment:
Has this been here before?

Reply:
It was not here before, but existed already. I tried to regroup methods to follow a more logical order